setup.py

```diff
@@ -34,24 +36,45 @@
 ]
 
 INSTALL_REQUIRES = [
-    'numpy >= 1.12',
-    'pandas >= 0.23.0, <1.0.0',
+    'matplotlib >= 3.0.0',
+    'numpy >= 1.15',
+    # exclude pandas==1.0.0 & 1.0.1 for GH142, and 0.24.0 for GH114
+    'pandas >= 0.23.0,!=0.24.0,!=1.0.0,!=1.0.1',
```

Comment:
Consider adding upper limits to these requirements (e.g. add `,<2.0.0` to pandas)

Reply:
Can we safely leave off these upper limits until Travis finds a problem with `upgrade-strategy=eager`?

Reply:
Eager upgrades for testing helps, but the problem is that we can't retroactively impose upper limits.  For example we release rdtools 2.0 today, pandas releases 2.0 tomorrow and breaks something, anyone who installs rdtools 2.0 will have a broken installation because we can only restrict the pandas version going forward.